RequestService/RequestService.Core/Services/JobService.cs

```diff
@@ -187,6 +187,11 @@ public async Task<bool> HasPermissionToViewRequestAsync(int requestID, int autho
 
             var userRoles = await _groupService.GetUserRoles(authorisedByUserID, cancellationToken);
 
+            if(userRoles.UserGroupRoles.Count==0)
+            {
+                return false;
+            }
+
```

Comment:
Do we also need to check for userRoles.UserGroupRoles.ContainsKey(referringGroupId), or does it handle that smoothly already?